http_server/controllers/utils.go

```diff
@@ -38,6 +39,18 @@ func (serverBot *ServerBot) SendBotMessage(msg string, link string, chatId int64
 	message := tgbotapi.NewMessage(chatId, msg)
 	message.ParseMode = "Markdown"
 	message.ReplyMarkup = startButton
+	return message
+}
+
+func ConstructSignupMessage(msg string, chatId int64) tgbotapi.MessageConfig {
+	button := service.GetHelperButtons()
+	message := tgbotapi.NewMessage(chatId, msg)
+	message.ParseMode = "Markdown"
+	message.ReplyMarkup = button
+	return message
+}
+
+func (serverBot *ServerBot) SendBotMessage(message tgbotapi.MessageConfig) bool {
 	if _, err := serverBot.bot.Send(message); err != nil {
 		log.Println(err)
```

Comment:
add some prefix maybe, e.g. log.Printf("SendBotMessage error: %s\n", err.Error()), easier to locate